<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLSegmentationGeometryWidget</class>
 <widget class="qMRMLWidget" name="qMRMLSegmentationGeometryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SegmentationGeometryWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_SourceGeometry">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_SourceGeometryNode">
        <property name="text">
         <string>Source geometry:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_SourceGeometryNode">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
          <string>vtkMRMLSegmentationNode</string>
          <string>vtkMRMLModelNode</string>
          <string>vtkMRMLMarkupsROINode</string>
          <string>vtkMRMLAnnotationROINode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="noneDisplay">
         <string>Showing current segmentation labelmap geometry. Click here to change</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_VolumeSpacingOptions">
        <property name="title">
         <string>Volume spacing options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <property name="horizontalSpacing">
          <number>4</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_OversamplingFactor">
           <property name="toolTip">
            <string>Split each voxel of the volume to this many voxels along each direction. Useful when increasing the resolution is needed</string>
           </property>
           <property name="text">
            <string>Oversampling factor:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ctkDoubleSpinBox" name="DoubleSpinBox_OversamplingFactor">
           <property name="toolTip">
            <string>Split each voxel of the volume to this many voxels along each direction. Useful when increasing the resolution is needed</string>
           </property>
           <property name="minimum">
            <double>0.050000000000000</double>
           </property>
           <property name="maximum">
            <double>20.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_IsotropicSpacing">
           <property name="toolTip">
            <string>Resample the volume to have isotropic spacing, which means the voxels will be cubes. Use smallest spacing. Useful if the volume has elongated voxels.</string>
           </property>
           <property name="text">
            <string>Isotropic spacing:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox_IsotropicSpacing">
           <property name="toolTip">
            <string>Resample the volume to have isotropic spacing, which means the voxels will be cubes. Use smallest spacing. Useful if the volume has elongated voxels.</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_SegmentationLabelmapGeometry">
     <property name="title">
      <string>Segmentation labelmap geometry</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="6" column="0">
       <widget class="QLabel" name="label_Directions">
        <property name="text">
         <string>Directions:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_Origin">
        <property name="text">
         <string>Origin:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_Spacing">
        <property name="text">
         <string>Spacing:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="ctkMatrixWidget" name="MatrixWidget_Directions">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <property name="rowCount">
         <number>3</number>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="qMRMLCoordinatesWidget" name="MRMLCoordinatesWidget_Origin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="quantity">
         <string>length</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_Dimensions">
        <property name="text">
         <string>Dimensions:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLCoordinatesWidget" name="MRMLCoordinatesWidget_Spacing">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="decimalsOption">
         <set>ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts|ctkDoubleSpinBox::DecimalsByValue|ctkDoubleSpinBox::InsertDecimals|ctkDoubleSpinBox::ReplaceDecimals</set>
        </property>
        <property name="quantity">
         <string>length</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLCoordinatesWidget" name="MRMLCoordinatesWidget_Dimensions">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_Error">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Error message</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCoordinatesWidget</class>
   <extends>ctkCoordinatesWidget</extends>
   <header>qMRMLCoordinatesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header>ctkCoordinatesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkMatrixWidget</class>
   <extends>QWidget</extends>
   <header>ctkMatrixWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qMRMLSegmentationGeometryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_SourceGeometryNode</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>3</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qMRMLSegmentationGeometryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLCoordinatesWidget_Dimensions</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>3</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qMRMLSegmentationGeometryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLCoordinatesWidget_Spacing</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>5</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qMRMLSegmentationGeometryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLCoordinatesWidget_Origin</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
